src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenExpression.vb

```diff
@@ -0,0 +1,519 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Roslyn.Test.Utilities
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
+    Public Class CodeGenExpression
+        Inherits BasicTestBase
+
+        <Fact, WorkItem(25692, "https://github.com/dotnet/roslyn/issues/25692")>
+        Public Sub CIntFix_01()
+            Dim source =
+<compilation>
+    <file name="a.vb">
+Option Strict On
+
+Imports System
+Imports Microsoft.VisualBasic
+
+Class C1
+    Public Shared Function SingleToSByte(number as Single) As SByte
+        Return CSByte(Fix(number))
```

Comment:
>Fix(number) [](start = 22, length = 11)

It looks like these tests cover only explicit conversions. I think we also need to have tests for implicit conversions applied to a Fix calls.